snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -121,6 +123,8 @@
             );
         }
         else {
+            document.getElementById('snippet_set').dataset.userCountry = '';
```

Comment:
I'm guessing it's expected that this could result in an empty query param value for `country` in the stats ping?